Minio.Functional.Tests/Minio.Functional.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
You replaced `net7.0` with `net8.0` here.
I guess that is because `net7.0` is going to be EOL on May 14th, 2024, which is less than a month away as of today.
So, it makes sense. 
However, check the below change in `Minio/Minio.csproj` file; `net8.0` is added instead of replacing `net7.0`.
We have to be consistent all over this PR.
I think removing `net7.0` is the better choice.